.circleci/config.yml

```
@@ -574,12 +574,13 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - dependencies-osx-{{ .Branch }}-{{ checksum ".circleci/config.yml" }}-{{ checksum ".circleci/osx_install_dependencies.sh" }}
+            - dependencies-osx-{{ checksum ".circleci/osx_install_dependencies.sh" }}
+      # DO NOT EDIT between here and save_cache, but rather edit ./circleci/osx_install_dependencies.sh
```

Maybe just suggest to invalidate cache manually?
```suggestion
      # WARNING! Remember to invalidate cache manually if you edit anything between this comment and save_cache
```

---

Well, it's better to use the scripts to keep us CI system agnostic anyways...

---

And it should be implicit that that's the underlying reason for the other comment...

---

I could add this as a second line of comment, though, if you want.

---

Well, ok. I'm fine with either version, I'd just emphasize the need to invalidate. But focusing on redirecting people to the script might be good too.

---

Now we have both ;-).